ibtk/src/solvers/impls/SCPoissonPointRelaxationFACOperator.cpp

```diff
@@ -74,36 +74,37 @@
 
 // FORTRAN ROUTINES
 #if (NDIM == 2)
-#define GS_SMOOTH_FC IBTK_FC_FUNC(gssmooth2d, GSSMOOTH2D)
+#define CONST_DC_GS_SMOOTH_FC IBTK_FC_FUNC(const_dc_gssmooth2d, CONST_DC_GSSMOOTH2D)
```

Comment:
Let's stick with the original names for the original functions. Although unlikely, it is possible that they are used in other places.

Reply:
@boyceg: This will cause confusion. The renaming was done to make function context clearer. There was various combinations of var C, const C, var D, const D.

Reply:
I also like the renames and I don't see any other usages of `gssmooth2d` in the library. Since the function prototypes are in a source file and not a header these functions are de facto internal anyway.

Reply:
OK, well, if they are getting renamed, let's put "GS smooth" in the front instead of the back.

Reply:
Also, suggest to rename "alpha" and "beta" to "D" and "C" (or is it "C" and "D") in the list of arguments both in the C prototype and in the Fortran routines.

Reply:
I'm fine either way.

Reply:
My usual rationale in naming is to go from "more general" to "more specific" for these long function names. I guess one could argue that it would better follow that pattern to write `smooth_gs...`...

Reply:
@Raamkrishnan can change the names (`smooth_gs` as prefix) along with my other suggestions. He can also add the changelog entry.